### DISCUSSION TITLE
Docker environment variables

### DISCUSSION THREAD

**Discussion by brplate** [opensourcepos/opensourcepos#4101] [Q&A]
Where can I get a list of supported docker environment variables to use with OSPOS?

**Comment by jekkos** [comment]
Hi most of these can be found in our docker compose files. So.I would recommend using those to spin up a new stack.

**Reply by jekkos** [reply]
-CI_ENVIRONMENT=production
          - FORCE_HTTPS=false
          - PHP_TIMEZONE=UTC
          - MYSQL_USERNAME=admin
          - MYSQL_PASSWORD=pointofsale
          - MYSQL_DB_NAME=ospos
          - MYSQL_HOST_NAME=mysql
